### DISCUSSION TITLE
Welcome to try Trame-based code: SpinView

### DISCUSSION THREAD

**Discussion by MXJK851** [Kitware/trame#351] [General]
Hi everyone,

Welcome to using my new trame-based code SpinView: https://github.com/MXJK851/SpinView.  (It would be great if you could star this project. :grin:)

**@trameteam**  Thank you very much for all the excellent work in this fantastic framework, and I really like using it.

I am wondering if SpinView could have a link or broadcast on your website like what we have on our homepage.  :blush: https://mxjk851.github.io/SpinView/
 
Best,
Qichen
PhD candidate, KTH Royal Institute of Technology

**Comment by jourdain** [comment]
FYI https://www.linkedin.com/posts/sebastienjourdain_trame-spinview-activity-7114654056345698304-nc9P?utm_source=share&utm_medium=member_desktop
